data/groups.json

```diff
@@ -78,22 +78,19 @@
       "cad789",
       "cryptostorms",
       "dafricash",
-      "devasive",
       "doweig",
       "cannabananadelegate",
       "diplo",
       "djselery",
       "fixcrypt",
       "grexx",
       "hoop",
-      "samuray",
```

Comment:
Source?

Reply:
https://github.com/vekexasia/dpos-tools-data/issues/172

Reply:
Hello!
I am no longer part of AFT for 1 year and 3 months now so I talked to vekexasia to change my tag on Liskbuilders site and he told me to open a github issue at https://github.com/vekexasia/dpos-tools-data/issues .